azure-pipelines.yml

```diff
@@ -485,51 +491,47 @@ jobs:
     artifact: chip_earlgrey_cw310_hyperdebug-build-out
     displayName: Upload artifacts for CW310
     condition: failed()
-
-- job: chip_earlgrey_cw310_splice_rom
-  displayName: Splice ROM into CW310 bitstream
-  dependsOn:
-    - chip_earlgrey_cw310
-    - chip_earlgrey_cw310_hyperdebug
-    - sw_build
-  condition: |
-    and
-    (
-      succeeded('chip_earlgrey_cw310'),
-      succeeded('sw_build'),
-      in(dependencies.chip_earlgrey_cw310_hyperdebug.result, 'Succeeded', 'SucceededWithIssues', 'Skipped')
-    )
+- job: chip_englishbreakfast_cw305
+  displayName: CW305's Bitstream
+  # Build CW305 variant of the English Breakfast toplevel design using Vivado
+  dependsOn: build_and_execute_verilated_tests_englishbreakfast
+  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyCdcChanges'], '0'))
   pool: ci-public
-  timeoutInMinutes: 10
+  timeoutInMinutes: 120 # 2 hours
   steps:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
   - template: ci/download-artifacts-template.yml
     parameters:
       downloadPartialBuildBinFrom:
-        - chip_earlgrey_cw310
-        - sw_build
-        - ${{ if eq(variables['Build.SourceBranchName'], 'master') }}:
-          - chip_earlgrey_cw310_hyperdebug
+        - build_and_execute_verilated_tests_englishbreakfast
   - bash: |
       set -e
-      . util/build_consts.sh
-
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-
-      util/fpga/splice_rom.sh -t cw310 -T earlgrey -b PROD
-
-    displayName: Splicing bitstream with Vivado
+      ci/scripts/build-bitstream-vivado.sh top_englishbreakfast cw305
```

Comment:
I'm currently zoomed into the first commit — isn't this the script we were going to remove?  Or were we only planning to remove it for earlgrey?

Reply:
We don't currently have a bazel rule for englishbreakfast. This will ultimately need to be resolved by a more comprehensive multi-top solution, but for now, we won't remove the shell scripts entirely.